benchmarks/gravity_mantle/mantle_gravity.prm

```diff
@@ -0,0 +1,87 @@
+# A simple setup for testing the gravity postprocessor using the
+# sample scheme 'map'.
+# See the corresponding section in the manual for more information.
+
+# General parameters
+set Dimension                              = 3
+set End time                               = 0
+set Output directory                       = output_shellgravity_S40RTS_new
```

Comment:
please replace following our usual standard `output-gravity-mantle` or similar (at least remove the _new).